packager/media/event/combined_muxer_listener.cc

```diff
@@ -0,0 +1,72 @@
+// Copyright 2014 Google Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+#include "packager/media/event/combined_muxer_listener.h"
+
+namespace shaka {
+namespace media {
+
+CombinedMuxerListener::CombinedMuxerListener(std::list<std::unique_ptr<MuxerListener>>* muxer_listeners) {
+    DCHECK(muxer_listeners);
```

Comment:
Two spaces indent: https://google.github.io/styleguide/cppguide.html#Spaces_vs._Tabs

We generally follow Chromium C++ style guide, which is derived from Google C++ style guide with some exceptions: https://chromium.googlesource.com/chromium/src/+/master/styleguide/c++/c++.md

You may use clang-format to format the code for you. If you have git-clang-format installed, config clang-format to Chromium style:

```shell
$ git config --global clangformat.style Chromium
```

Run clang format against a base CL
```shell
$ git clang-format <base commit>
```